.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Pipeline
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  Metamodel-Modeling-Foundations:
+    uses:  MDSD-Tools/Palladio-Build-ActionsPipeline/.github/workflows/build.yml@v3.2
```

Comment:
Also, there is noch Palladio-Build-ActionsPipeline repository in MDSD-Tools. Here you should use the pipeline that is also used in the Palladio organization.
Just change MDSD-Tools to PalladioSimulator

Reply:
done